src/Lykke.SettingsReader/ReloadingManager/LocalSettingsReloadingManager.cs

```diff
@@ -1,10 +1,11 @@
 ﻿using System;
 using System.IO;
 using System.Threading.Tasks;
+using Lykke.SettingsReader.ReloadingManager.Configuration;
 
 namespace Lykke.SettingsReader
 {
-    public class LocalSettingsReloadingManager<TSettings> : ReloadingManagerBase<TSettings>
+    public class LocalSettingsReloadingManager<TSettings> : ReloadingManagerWithConfigurationBase<TSettings>
```

Comment:
Will be nice to mark this class as [PublicAPI], as well as anothe public classes/interfaces

Reply:
Same as for NulNull - decided not to add a new reference to JetBrains nuget